### PR TITLE
Change dockerfile to python3

### LINKED ISSUE #4772: Docker doesn't use Python3
**Describe the bug**
Docker still uses python3

**To Reproduce**
Build the image with the dockerfile

**Expected behavior**
It should use python3

**Additional context**
Here is the linecode for it.
https://github.com/fossasia/open-event-server/blob/eaffb251c38848f5d7d78a52cde92ff23761b947/Dockerfile#L14

### PR DESCRIPTION
**Pull request by Kreijstal**
Now it adds python3 statements, this should solve #4772

Fixes #4772

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
It resolves the issue of docker still using python2.x

#### Changes proposed in this pull request:

- change from python to python3
- change from pip to pip3
-

### CODE REVIEW

**Comment by SaptakS** [comment]
@Kreijstal have you run the code? There is no python2 dependencies which might fail by pip3 install, right?

File: Dockerfile [hunk 1]
```diff
@@ -11,15 +11,15 @@ RUN apt-get update && apt-get install -y wget git ca-certificates curl && update
 
 
 # install deps
-RUN apt-get install -y --no-install-recommends build-essential python-dev libpq-dev libevent-dev libmagic-dev && apt-get clean -y
+RUN apt-get install -y --no-install-recommends build-essential python3-dev libpq-dev libevent-dev libmagic-dev && apt-get clean -y
```
Comment (schedutron) [inline Dockerfile]: Yes, is this successful locally?

**Comment by Kreijstal** [comment]
works on my machine

File: Dockerfile (hunk 1 above)
Comment (Kreijstal) [reply Dockerfile]: it installs successfully

**Comment by Kreijstal** [comment]
I think that would take considerably way longer, in that case I will provide a PR when I fix all the docker issues.

**Pull request closed without merging**